documentation/src/main/asciidoc/topics/ref_rest_xsite.adoc

```diff
@@ -24,26 +24,26 @@ the following example:
 |Value |Description
 
 |`online`
-|All nodes in the backup site are online
+| All nodes in the backup location are online.
```

Comment:
this is incorrect (even the old docs). It should be:
`All nodes in the local site see the remote site online` (or similar)
It needs to be fixed for offline and mixed.
For mixed:
Some nodes in the local site see the remote site online and other see the remote site offline.